src/model/control_desk.py

```diff
@@ -1,37 +1,39 @@
+# coding=utf-8
+"""models for X-Touch"""
 from abc import ABC, abstractmethod
-from Network import NetworkManager
-from typing import Tuple
 from uuid import uuid4
 
 import proto.Console_pb2
+from Network import NetworkManager
+from model.color_hsi import ColorHSI
 
 
 def _generate_unique_id() -> str:
     return str(uuid4())
 
 
 class DeskColumn(ABC):
+    """TODO docstring"""
```

Comment:
"""This class represents a single column inside a bank.

This class should not be instantiated directly. Instead the implementing classes ColorDeskColumn and RawDeskColumn should be used.

TODO: Also implement U,UA and A types.
"""